app/models/community_resource.rb

```diff
@@ -31,20 +33,48 @@ def self.published
     )
   end
 
+  def title; description; end
+  def self.matchable; published; end
+
   def published?
     now = Time.current
     is_approved &&
       (publish_from.present? ? publish_from <= now : true) &&
       (publish_until.nil? || now < publish_until)
   end
 
+  def categories_for_tags
+    Category.where(name: tag_list)
+  end
+
   def all_tags_unique
     all_tags_list.flatten.map(&:downcase).uniq
   end
 
   def all_tags_to_s
     all_tags_unique.join(', ')
   end
+
+  def preferred_contact_method
+    # TODO: This is a hack that makes things work for now
+    # The test creates a contact method that will match this
+    # and the dev db seeding creates a couple, too
+    ContactMethod.method_name('call').last
```

Comment:
Is there some way I can fix this in its own PR? In other codebases I work in, this whole workflow is something I'd hide behind a feature flag so I can get this PR out first, and then get the contact methods for community resources working in a separate PR. I could also try to fix it here, especially if anyone's willing to help me with it. Please let me know

Reply:
Added an issue for this at https://github.com/rubyforgood/mutual-aid/issues/986